_projects/SP19-CITL_VR_Lecture

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: CITL VR Lecture
+image: <link to project image (optional)>
```

Comment:
```suggestion
```

Reply:
Hi Shan, I do not know how to edit the file. Do I need to delete it and make a new one?

Reply:
You can make changes on your branch, they will be reflected here.